microcosm_daemon/daemon.py

```diff
@@ -78,6 +78,10 @@ def root_path(self):
         """
         return None
 
+    @property
+    def initial_state(self):
```

Comment:
We need a way to cleanly override the initial state of a daemon without changing its base behavior.